web/vtadmin/src/components/charts/TabletQPSChart.tsx

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2021 The Vitess Authors.
+ * Copyright 2024 The Vitess Authors.
```

Comment:
I think this is fine, but we don't typically NEED to change the year when updating an existing file.

Reply:
+1

Reply:
Ah, that's good to know. I thought someone had told me that we needed to in the past (it may have just been for net-new files). I won't update them going forwards.